cmd/build/v2/services.go

```diff
@@ -107,6 +107,8 @@ func getToBuildTag(manifestName, svcName string, b *model.BuildInfo) string {
 		return b.Image
 	case (shouldBuildFromDockerfile(b) && shouldAddVolumeMounts(b)) || shouldAddVolumeMounts(b):
 		return fmt.Sprintf("%s/%s-%s:%s", targetRegistry, manifestName, svcName, model.OktetoImageTagWithVolumes)
+	case shouldBuildFromDockerfile(b) && b.Image != "":
```

Comment:
Sounds weird having to check if it has to be built from a dockerfile but then, return the Image from b. Would it make sense to switch the case clauses?